package.json

```diff
@@ -32,6 +32,7 @@
     "extend": "^3.0.1",
     "gcp-metadata": "^0.6.1",
     "nan": "^2.8.0",
+    "numeral": "^2.0.6",
```

Comment:
I'd avoid dependencies that are not crucial, to minimize the profiler footprint.

Reply:
done.